src/foam/u2/view/NestedTreeView.js

```diff
@@ -54,7 +54,19 @@ foam.CLASS({
       if ( this.translationService ) {
         labelString = self.translationService.getTranslation(foam.locale, self.data.label, self.data.label);
       }
+
```

Comment:
Why does this have to be here? NestedTreeView isn't Menu specific and we don't want to have to make it tightly coupled to menus or even a specific menu handler. The idea is that we should be able to make menu handlers extensible.

Reply:
due to its forced styling i couldnt do it with any other way, check : https://github.com/kgrgreer/foam3/pull/3930 where i used row config to handle it, it shows the seprator, but there is forced stlying on it which is not logical, due to its being clickable and had uneeded paddings.